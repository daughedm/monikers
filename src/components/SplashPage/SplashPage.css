.splash-page {
  background-image: url('../../assets/monikers_furry_two-02.svg');
  background-size: 100%;
  background-repeat: no-repeat;
  background-position: right 50% bottom;
  height: 100vh;
  animation-name: tiger;
  animation-duration: 10s;
}

@keyframes tiger {
  0%   {
    background-image: url('../../assets/monikers_furry_two-02.svg');
  }
  25%  {
    background-image: url('../../assets/monikers_furry_two-02.svg');
    
  }
  50%  {
    background-image: url('../../assets/monikers_furry_two-02.svg');
    
  }
  /* 75%  {
    background-image: url('../../assets/monikers_furry_two-02.svg');
    
  } */
  100% {
    background-image: url('../../assets/monikers_furry_one-01.svg');
    
  }
}

.button-wrapper {
  width: 75%;
  margin: auto;
  padding: 0;
}

.logo-lockup {
  display: block;
  margin: auto;
  padding: 60px 0 40px 0;
  width: 65%;
}

.start-game-button {
  width: 100%;
  height: 50px;
  color: #FFF;
  background-color: rgba(76, 189, 159, 1);
  border: 1px solid #FFF;
  border-radius: 5px;
  font-size: 1rem;
  margin-top: 40px;
  -webkit-box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
  -moz-box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
  box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
  
}

.instructions-button {
  width: 100%;
  height: 50px;
  color: #FFF;
  background-color: rgba(134, 106, 173, 1);
  border: 1px solid #FFF;
  border-radius: 5px;
  font-size: 1rem;
  margin-top: 20px;
  -webkit-box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
  -moz-box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
  box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
  
}

.navlink {
  color: #FFF;
  text-decoration: none;
  padding: 15px 60px;
}

a {
  width: 100%;
}

