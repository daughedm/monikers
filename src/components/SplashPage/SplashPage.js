import React from 'react';
import { NavLink } from 'react-router-dom';
import logolockup from '../../assets/Monikers_logo_lockup-02.svg';
import './SplashPage.css';

const SplashPage = () => {
  return (
    <div className="background-furry">
      <div className="splash-page">
        <img className="logo-lockup" src={logolockup}/>
        <div className="button-wrapper" >
          <button className="start-game-button">
            <NavLink className="navlink" to="/setup">NEW GAME</NavLink>
          </button>
          <button className="instructions-button">
            <NavLink className="navlink" to="/instructions">HOW TO PLAY</NavLink>
          </button>
        </div>
      </div>
    </div>
  );
};

export default SplashPage;
