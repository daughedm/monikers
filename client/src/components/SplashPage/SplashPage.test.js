/* eslint-disable max-len */

import React from 'react';
import { shallow } from 'enzyme';
import SplashPage from './splashPage';

describe('splashPage', () => {
  let splashPage;

  beforeEach(() => {
    splashPage = shallow(<SplashPage />);
  });

  it('matches the snapshot', () => {
    expect(splashPage).toMatchSnapshot();
  });
});
