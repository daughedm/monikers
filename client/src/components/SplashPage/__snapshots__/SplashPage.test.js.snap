// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`splashPage matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SplashPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="splash-page"
>
        <img
                alt="logolockup"
                className="logo-lockup animated bounceIn"
                src="Monikers_logo_lockup-02.svg"
        />
        <div
                className="button-wrapper"
        >
                <button
                        className="start-game-button"
                >
                        <NavLink
                                activeClassName="active"
                                aria-current="page"
                                className="navlink"
                                to="/setup"
                        >
                                NEW GAME
                        </NavLink>
                </button>
                <button
                        className="instructions-button"
                >
                        <NavLink
                                activeClassName="active"
                                aria-current="page"
                                className="navlink"
                                to="/instructions"
                        >
                                HOW TO PLAY
                        </NavLink>
                </button>
                <button
                        className="buy-button"
                >
                        <a
                                className="navlink"
                                href="http://www.monikersgame.com/"
                        >
                                BUY THE GAME
                        </a>
                </button>
        </div>
</div>,
      "className": "background-furry",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt="logolockup"
            className="logo-lockup animated bounceIn"
            src="Monikers_logo_lockup-02.svg"
/>,
          <div
            className="button-wrapper"
>
            <button
                        className="start-game-button"
            >
                        <NavLink
                                    activeClassName="active"
                                    aria-current="page"
                                    className="navlink"
                                    to="/setup"
                        >
                                    NEW GAME
                        </NavLink>
            </button>
            <button
                        className="instructions-button"
            >
                        <NavLink
                                    activeClassName="active"
                                    aria-current="page"
                                    className="navlink"
                                    to="/instructions"
                        >
                                    HOW TO PLAY
                        </NavLink>
            </button>
            <button
                        className="buy-button"
            >
                        <a
                                    className="navlink"
                                    href="http://www.monikersgame.com/"
                        >
                                    BUY THE GAME
                        </a>
            </button>
</div>,
        ],
        "className": "splash-page",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "logolockup",
            "className": "logo-lockup animated bounceIn",
            "src": "Monikers_logo_lockup-02.svg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <button
                className="start-game-button"
>
                <NavLink
                                activeClassName="active"
                                aria-current="page"
                                className="navlink"
                                to="/setup"
                >
                                NEW GAME
                </NavLink>
</button>,
              <button
                className="instructions-button"
>
                <NavLink
                                activeClassName="active"
                                aria-current="page"
                                className="navlink"
                                to="/instructions"
                >
                                HOW TO PLAY
                </NavLink>
</button>,
              <button
                className="buy-button"
>
                <a
                                className="navlink"
                                href="http://www.monikersgame.com/"
                >
                                BUY THE GAME
                </a>
</button>,
            ],
            "className": "button-wrapper",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <NavLink
                  activeClassName="active"
                  aria-current="page"
                  className="navlink"
                  to="/setup"
>
                  NEW GAME
</NavLink>,
                "className": "start-game-button",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "activeClassName": "active",
                  "aria-current": "page",
                  "children": "NEW GAME",
                  "className": "navlink",
                  "to": "/setup",
                },
                "ref": null,
                "rendered": "NEW GAME",
                "type": [Function],
              },
              "type": "button",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <NavLink
                  activeClassName="active"
                  aria-current="page"
                  className="navlink"
                  to="/instructions"
>
                  HOW TO PLAY
</NavLink>,
                "className": "instructions-button",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "activeClassName": "active",
                  "aria-current": "page",
                  "children": "HOW TO PLAY",
                  "className": "navlink",
                  "to": "/instructions",
                },
                "ref": null,
                "rendered": "HOW TO PLAY",
                "type": [Function],
              },
              "type": "button",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  className="navlink"
                  href="http://www.monikersgame.com/"
>
                  BUY THE GAME
</a>,
                "className": "buy-button",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "BUY THE GAME",
                  "className": "navlink",
                  "href": "http://www.monikersgame.com/",
                },
                "ref": null,
                "rendered": "BUY THE GAME",
                "type": "a",
              },
              "type": "button",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="splash-page"
>
          <img
                    alt="logolockup"
                    className="logo-lockup animated bounceIn"
                    src="Monikers_logo_lockup-02.svg"
          />
          <div
                    className="button-wrapper"
          >
                    <button
                              className="start-game-button"
                    >
                              <NavLink
                                        activeClassName="active"
                                        aria-current="page"
                                        className="navlink"
                                        to="/setup"
                              >
                                        NEW GAME
                              </NavLink>
                    </button>
                    <button
                              className="instructions-button"
                    >
                              <NavLink
                                        activeClassName="active"
                                        aria-current="page"
                                        className="navlink"
                                        to="/instructions"
                              >
                                        HOW TO PLAY
                              </NavLink>
                    </button>
                    <button
                              className="buy-button"
                    >
                              <a
                                        className="navlink"
                                        href="http://www.monikersgame.com/"
                              >
                                        BUY THE GAME
                              </a>
                    </button>
          </div>
</div>,
        "className": "background-furry",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="logolockup"
              className="logo-lockup animated bounceIn"
              src="Monikers_logo_lockup-02.svg"
/>,
            <div
              className="button-wrapper"
>
              <button
                            className="start-game-button"
              >
                            <NavLink
                                          activeClassName="active"
                                          aria-current="page"
                                          className="navlink"
                                          to="/setup"
                            >
                                          NEW GAME
                            </NavLink>
              </button>
              <button
                            className="instructions-button"
              >
                            <NavLink
                                          activeClassName="active"
                                          aria-current="page"
                                          className="navlink"
                                          to="/instructions"
                            >
                                          HOW TO PLAY
                            </NavLink>
              </button>
              <button
                            className="buy-button"
              >
                            <a
                                          className="navlink"
                                          href="http://www.monikersgame.com/"
                            >
                                          BUY THE GAME
                            </a>
              </button>
</div>,
          ],
          "className": "splash-page",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "logolockup",
              "className": "logo-lockup animated bounceIn",
              "src": "Monikers_logo_lockup-02.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <button
                  className="start-game-button"
>
                  <NavLink
                                    activeClassName="active"
                                    aria-current="page"
                                    className="navlink"
                                    to="/setup"
                  >
                                    NEW GAME
                  </NavLink>
</button>,
                <button
                  className="instructions-button"
>
                  <NavLink
                                    activeClassName="active"
                                    aria-current="page"
                                    className="navlink"
                                    to="/instructions"
                  >
                                    HOW TO PLAY
                  </NavLink>
</button>,
                <button
                  className="buy-button"
>
                  <a
                                    className="navlink"
                                    href="http://www.monikersgame.com/"
                  >
                                    BUY THE GAME
                  </a>
</button>,
              ],
              "className": "button-wrapper",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <NavLink
                    activeClassName="active"
                    aria-current="page"
                    className="navlink"
                    to="/setup"
>
                    NEW GAME
</NavLink>,
                  "className": "start-game-button",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "activeClassName": "active",
                    "aria-current": "page",
                    "children": "NEW GAME",
                    "className": "navlink",
                    "to": "/setup",
                  },
                  "ref": null,
                  "rendered": "NEW GAME",
                  "type": [Function],
                },
                "type": "button",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <NavLink
                    activeClassName="active"
                    aria-current="page"
                    className="navlink"
                    to="/instructions"
>
                    HOW TO PLAY
</NavLink>,
                  "className": "instructions-button",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "activeClassName": "active",
                    "aria-current": "page",
                    "children": "HOW TO PLAY",
                    "className": "navlink",
                    "to": "/instructions",
                  },
                  "ref": null,
                  "rendered": "HOW TO PLAY",
                  "type": [Function],
                },
                "type": "button",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <a
                    className="navlink"
                    href="http://www.monikersgame.com/"
>
                    BUY THE GAME
</a>,
                  "className": "buy-button",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "BUY THE GAME",
                    "className": "navlink",
                    "href": "http://www.monikersgame.com/",
                  },
                  "ref": null,
                  "rendered": "BUY THE GAME",
                  "type": "a",
                },
                "type": "button",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
